yarn-project/circuits.js/src/structs/constants.ts

```diff
@@ -41,5 +41,7 @@ export const ROLLUP_VK_TREE_HEIGHT = 8;
 
 export const FUNCTION_SELECTOR_NUM_BYTES = 4;
 
+export const MAPPING_SLOT_PEDERSEN_SEPARATOR = 4;
```

Comment:
is there a place perhaps from cpp/bberg from where we are getting this value from? Maybe add a link to that there if in future we change this

Reply:
is from the constants.hpp file that this file already shadows